### PR TITLE
Packages minimal

### PR DESCRIPTION
**Pull request by eboschtwonav**

### CODE REVIEW

File: make_var_mx6ul_dart_debian.sh [hunk 1]
```diff
@@ -132,6 +132,7 @@ function usage() {
 	echo "       all         		-- build or rebuild kernel/bootloader/rootfs"
 	echo "       bootloader  		-- build or rebuild bootloader (u-boot+SPL)"
 	echo "       kernel      		-- build or rebuild linux kernel for this board"
+	echo "       package      		-- build or linux kernel package for this board"
```
Comment (tpaschidis) [inline make_var_mx6ul_dart_debian.sh]: -- build or rebuild linux kernel package for this board"

**Review by tpaschidis — DISMISSED** [review]

File: make_var_mx6ul_dart_debian.sh (hunk 1 above)
Comment (eboschtwonav) [reply make_var_mx6ul_dart_debian.sh]: fixed!

File: make_var_mx6ul_dart_debian.sh [hunk 2]
```diff
@@ -563,8 +564,8 @@ rm -f user-stage
 	install -m 0644 ${G_VARISCITE_PATH}/hostapd.conf ${ROOTFS_BASE}/etc/
 	install -m 0755 ${G_TWONAV_PATH}/rc_files/rc.local ${ROOTFS_BASE}/etc/
 	install -m 0755 ${G_TWONAV_PATH}/rc_files/rc.installer ${ROOTFS_BASE}/etc/
-	install -m 0644 ${G_VARISCITE_PATH}/splash.bmp ${ROOTFS_BASE}/boot/
-	install -m 0644 ${G_VARISCITE_PATH}/uuid.h ${ROOTFS_BASE}/usr/include/bluetooth/
+	#install -m 0644 ${G_VARISCITE_PATH}/splash.bmp ${ROOTFS_BASE}/boot/
+	#install -m 0644 ${G_VARISCITE_PATH}/uuid.h ${ROOTFS_BASE}/usr/include/bluetooth/
```
Comment (lluisdiaz) [inline make_var_mx6ul_dart_debian.sh]: we need bluetooth, so reactivate bluetooth!

File: variscite/variscite-bluetooth [hunk 3]
```diff
@@ -96,7 +96,9 @@ bt_found || exit 0
 		enable_bt
 
 		# Get BT MAC address
-		BT_MACADDR=$(get_bt_macaddr)
+		#BT_MACADDR=$(get_bt_macaddr)
+		# Provisional fix to fake max address
+		BT_MACADDR="de:ad:be:ef:ca:fe"
```
Comment (lluisdiaz) [inline variscite/variscite-bluetooth:101]: This should be documented somewhere if it's a provsional fix.

Comment (eboschtwonav) [reply variscite/variscite-bluetooth:101]: We will remove all about variscite wifi/bluetooth in a few days, so I don't think that we need to document this issue

**Review by lluisdiaz — APPROVED** [review]

**Review by tpaschidis — APPROVED** [review]

**Pull request merged**